lib/logging/monitor.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Logging
+  class Monitor < ::ZeroSilentFailures::Monitor
+    def initialize(service)
+      @service = service
+      super(@service)
+    end
+
+    ##
+    # log GET request
+    #
+    # @param message [String]
+    # @param metric [String]
+    # @param form_type [String]
+    # @param user_account_uuid [User]
+    #
+    def track_request(message, metric, form_type = 'Unknown Form Type', user_account_uuid = nil, call_location: nil)
+      function, file, line = parse_caller(call_location)
+
+      StatsD.increment(metric)
```

Comment:
Should we leave the `metric` to whatever the team wants or should we add a bit more structure so they can only pass in the prefix?
i.e. `{STATS_KEY}.error`

Reply:
For these functions I would let the metric be passed to the function.  Also, possibly the 'log level' [info, warn, error] unless you want to split out or specify that it will log as an error in the function name.